pkgs/development/tools/continuous-integration/buildkite-agent/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, makeWrapper, coreutils, git, openssh, bash, gnused, gnugrep }:
+
+stdenv.mkDerivation rec {
+  name = "buildkite-agent-2.1.8";
+
+  src = fetchurl {
+    url = "https://github.com/buildkite/agent/releases/download/v2.1.8/buildkite-agent-linux-386-2.1.8.tar.gz";
+    sha256 = "f54ca7da4379180700f5038779a7cbb1cef31d49f4a06c42702d68c34387c242";
+  };
+
+  buildInputs = [ makeWrapper ];
+  phases = "unpackPhase installPhase";
```

Comment:
I'm guessing that what you really want is to disable the build phase. If so, please consider setting `dontBuild = true` or `buildPhase = ":"` instead.